src/components/tooltip/tooltip.module.scss

```diff
@@ -12,7 +12,16 @@
   width: max-content;
   z-index: 1;
 
-  .triangle {
+  .triangle_bottom {
+    border-color: $tooltip-background-color transparent transparent transparent;
+    border-style: solid;
+    border-width: 5px;
+    margin-top: -5px;
+    position: absolute;
+    top: 105%;
+    right: 43.6%;
```

Comment:
This seems fragile and relies on the specific length of the current title for conservation status -- could we compute this instead?

Reply:
I changed the value of right to 50% and added margin variable that is calculated based on the position of info icon.
![tooltip_length_of_text](https://github.com/alveusgg/extension/assets/147619232/7fd0d206-f8b4-444e-932a-d7dc1c413f59)

I also considered centering tooltip based on the info icon and having the triangle in the middle of tooltip instead of centering tooltip in the ambassador card and having the triangle offset by margin, as it is right now. Unsure if it looks better.
![centered_to_icon](https://github.com/alveusgg/extension/assets/147619232/b9af548e-77cc-4872-9ef8-cbdfb7f94895)

Reply:
I think the current approach you have is best, as we know we'll always be in-bounds of the viewport if we remain within the card